templates/vue/src/components/Lightbox/media/ActivityMedia/index.vue

```diff
@@ -98,6 +99,9 @@ export default {
     hasPrev() {
       return this.activeQuestionIndex !== 0
     },
+    performDyadNodeCheck() {
+      return wp.getCurrentUser().roles.includes("copilot") || !this.node.isDyad
```

Comment:
Are youths and copilots the same?

Reply:
to be honest, I don't know... this was copied from the implementation that Aidin gave me... this might be why we're seeing funny behaviours